css/main.scss

```diff
@@ -16,6 +16,24 @@ $grey-color: #828282;
 $grey-color-light: lighten($grey-color, 40%);
 $grey-color-dark: darken($grey-color, 25%);
 
+table {
+	border-collapse: collapse;
+	width: 100%;
+	text-align: left;
+	border: 1px solid #c6c9cc;
+	border-radius: 5px;
+ 	padding: 10px 20px;
+	th {
+    text-align: center;
+  }
```

Comment:
The formatting is a bit odd

Reply:
even for scss? I ran this into a css->scss converter, kinda trusting the output. The th block could use indentation though.

Reply:
I'm just talking about the closing curly brace and the misaligned "text-align" property here